howtos/documentation-guidelines.md

```diff
@@ -18,63 +18,125 @@ In the spirit of "Always Progress", if you are confident you know what change ne
+| `docs`     | Next                     | [/docs/](../docs/)                                                  |
+| `docs`     | 8.1, 8.0, 1.3, ...       | [/versioned_docs/version-\*/](../versioned_docs/)                   |
+| `optimize` | Next                     | [/optimize/](../optimize/)                                          |
+| `optimize` | 3.9.0, 3.8.0, 3.7.0, ... | [/optimize_versioned_docs/version-\*/](../optimize_versioned_docs/) |
+
+When edits are intended to apply to both the current version _and beyond_, they should be made in both the most recent versioned folder and the "Next" version folder.
+
+### Version alignment
+
+Specific Optimize versions are aligned with Camunda Platform versions as follows:
+
+| Camunda Platform version | Optimize version |
+| ------------------------ | ---------------- |
+| 8.1                      | 3.9.0            |
+| 8.0                      | 3.8.0            |
+| 1.3                      | 3.7.0            |
+
+## Sidebar navigation
+
+[Sidebar navigation](https://docusaurus.io/docs/sidebar) of the documentation is managed in the sidebars files. [Each instance of the documentation](#instances-docs-vs-optimize) has its own sidebars file:
+
+| Instance   | Version(s)               | Sidebars path                                                                             |
+| ---------- | ------------------------ | ----------------------------------------------------------------------------------------- |
+| `docs`     | Next                     | [/docs/sidebars.js](../docs/sidebars.js)                                                  |
+| `docs`     | 8.1, 8.0, 1.3, ...       | [/versioned_sidebars/version-\*-sidebars.json](../versioned_sidebars/)                    |
+| `optimize` | Next                     | [/optimize/sidebars.js](../optimize/sidebars.js)                                          |
+| `optimize` | 3.9.0, 3.8.0, 3.7.0, ... | [/optimize_versioned_sidebars/version-\*-sidebars.json/](../optimize_versioned_sidebars/) |
+
+### Sidebar items
+
+The large object in each sidebars file contains two different types of items:
+
+- Items within the same [documentation instance](#instances-docs-vs-optimize) are [linked by the path to the target .md file](https://github.com/camunda/camunda-platform-docs/blob/89993fbc1446c203324f38139ae7eb40e19b14ac/versioned_sidebars/version-8.1-sidebars.json#L5):
+  ```json
+  "guides/introduction-to-camunda",
+  ```
+- Items in the opposite [documentation instance](#instances-docs-vs-optimize) are [linked by an object containing the title and URL of the target document](https://github.com/camunda/camunda-platform-docs/blob/89993fbc1446c203324f38139ae7eb40e19b14ac/versioned_sidebars/version-8.1-sidebars.json#L331-L335):
+  ```json
+  {
+    "type": "link",
+    "label": "What is Optimize?",
+    "href": "/optimize/components/what-is-optimize/"
+  },
+  ```
+
+> **Note**
```

Comment:
GitHub has admonitions now 🥳 

<img width="1110" alt="image" src="https://user-images.githubusercontent.com/1627089/196290974-26a669eb-6f26-4bcd-bde8-8a9ab9cc41a8.png">

Reply:
You should tell the engineering channel 😄

Reply:
oof, [people are NOT HAPPY](https://github.com/community/community/discussions/16925) about this feature, especially (a) the fact that it requires English language and (b) it uses the `blockquote` element. 

I'm personally fine leaving them in here but let me know if you feel otherwise.

Reply:
Wow. You can leave it. I think it works nicely for us.